components/Header.tsx

```diff
@@ -1,56 +1,82 @@
+import { SearchIcon } from "@chakra-ui/icons";
 import {
   Box,
   Button,
   Container,
-  Flex,
+  FormControl,
+  Grid,
   Heading,
-  Spacer,
-  useDisclosure,
+  HStack,
+  Input,
+  InputGroup,
+  InputRightElement,
 } from "@chakra-ui/react";
-import { useLogout, useUserQuery } from "@lib/hooks/user";
-import { useState } from "react";
-import { v4 as uuid } from "uuid";
-import LoginModal from "./LoginModal";
-
-export default function Header() {
-  const { isAuthenticated } = useUserQuery();
-  const { isOpen, onClose, onOpen } = useDisclosure();
-
-  const [loginModalKey, setLoginModalKey] = useState("");
-  const handleLoginClick = () => {
-    // Reset login modal each time it is opened
-    setLoginModalKey(uuid());
-    onOpen();
-  };
-
-  const logout = useLogout();
+import Link from "next/link";
+import HeaderUserInfo from "./HeaderUserInfo";
 
+const Header: React.FC = ({ children }) => {
   return (
-    <Box as="header" backgroundColor="secondaryBackground">
-      <Container py={5} maxWidth="container.md">
-        <Flex alignItems="baseline" justifyContent="space-between">
-          <Heading as="h1" size="md">
-            Book Bazar
+    <Box as="header" backgroundColor="secondaryBackground" boxShadow="md">
+      <Container py={{ base: 5, md: 8 }} maxWidth="container.lg">
+        <Grid
```

Comment:
One more thing that I think should be in this row (that didn't get included in the Figma unfortunately) is a button to go to the "create a textbook posting" wizard. As well, should we include some sort of toggle to add/remove the search bar from this, since I think we planned to put the search bar in the center for the screen for the home page?